source/common/common/utility.cc

```diff
@@ -38,6 +38,21 @@ using UnsignedMilliseconds = std::chrono::duration<uint64_t, std::milli>;
 
 } // namespace
 
+const std::string errorDetails(int error_code) {
+#ifdef WIN32
+  std::string buffer(128, '\0');
```

Comment:
nit: do you need to pre-allocate the buffer? can you pass null length and let the function do the allocation for you?

Reply:
using the flag `FORMAT_MESSAGE_ALLOCATE_BUFFER `

Reply:
yep, will do

/wait

Reply:
Done, see changes